src/opera/pge/base_pge.py

```diff
@@ -203,14 +206,40 @@ class and its inheritors, and as such, this class assumes access to the
     def __init__(self):
         self.name = None
         self.logger = None
+        self.runconfig_path = None
+        self.runconfig = RunConfig(self.runconfig_path)
```

Comment:
`PostProcessorMixin` should not have to define its own `runconfig` attribute since it `PgeExecutor` defines its own. Lines 209 and 210 should be removed. References to `self.runconfig` in `_create_catalog_metadata` should still work and reference the RunConfig created by `PgeExecutor`

Reply:
Done